getscreenmedia.js

```diff
@@ -6,11 +6,17 @@ module.exports = function (constraints, cb) {
     var error;
 
     if (adapter.browserDetails.browser === 'chrome') {
-        // check that the extension is installed by looking for a
-        // sessionStorage variable that contains the extension id
-        // this has to be set after installation unless the content
-        // script does that
-        if (sessionStorage.getScreenMediaJSExtensionId) {
+        if ('getDisplayMedia' in window.navigator) { // prefer spec getDisplayMedia
+            window.navigator.getDisplayMedia({video: true}).then(function (stream) {
+                callback(null, stream);
+            }).catch(function (err) {
+                callback(err);
```

Comment:
We should move this to promise based as well.

Reply:
I actually hope to retire this module soon (TM)